scripts/generate_logit_filters.sh

```diff
@@ -2,8 +2,12 @@
 
 set -euo pipefail
 
+echo "********************************* Start copy below this line ********************************************"
+
 echo "filter {"
 sed 's/^/  /' < config/logit/10_base.conf
 sed 's/^/  /' < config/logit/20_logsearch_for_cloudfoundry.conf
 sed 's/^/  /' < config/logit/30_custom_filters.conf
```

Comment:
We need to document/automate/script the generation of the contents of these files. When the source (logsearch-for-cloudfoundry) changes we will not get the updates and need to know how to regenerate these.

Reply:
Ah I misunderstood.